README.md

```diff
@@ -65,3 +70,11 @@ You will need to identify the Board ID. See the following url for details on how
 
 You will also need to get the lists on the board. See the following url for details on how to find this: https://developer.atlassian.com/cloud/trello/rest/api-group-boards/#api-boards-id-lists-get
 
+## Testing
+
+### Unit Tests
```

Comment:
It would be nice to include a one-liner to explain how users can run the tests (e.g. `poetry run pytest`) - particularly since we'll later want to expand to automated pipelines which will run the tests in this fashion

We should also specify any dependencies, e.g. Edge & its driver

Reply:
README updated.